src/System.Reflection.Metadata/src/System/Reflection/Internal/MemoryBlocks/MemoryMappedFileBlock.cs

```diff
@@ -24,22 +25,19 @@ internal unsafe MemoryMappedFileBlock(IDisposable accessor, SafeBuffer safeBuffe
 
         ~MemoryMappedFileBlock()
         {
-            Dispose(false);
+            DisposeImpl();
         }
 
-        protected override void Dispose(bool disposing)
+        public override void Dispose()
         {
-            if (_safeBuffer != null)
-            {
-                _safeBuffer.ReleasePointer();
-                _safeBuffer = null;
-            }
-
-            if (_accessor != null)
-            {
-                _accessor.Dispose();
-                _accessor = null;
-            }
+            DisposeImpl();
```

Comment:
I don't see the value in deviating from the FDG pattern of Dispose(bool). Following the pattern means I need less context when reading a method. Dispose(bool) has well known meaning across entire fx, DisposeImpl() does not.